Description: Use of unshift in gemspec
---

Index: ruby-sass-rails/sass-rails.gemspec
===================================================================
--- ruby-sass-rails.orig/sass-rails.gemspec
+++ ruby-sass-rails/sass-rails.gemspec
@@ -1,5 +1,5 @@
 # -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
+$:.unshift File.expand_path("../lib", __FILE__)
 require "sass/rails/version"
 
 Gem::Specification.new do |s|
